libretro-buildbot-recipe.sh

```diff
@@ -238,6 +238,18 @@ buildbot_log() {
 	fi
 }
 
+buildbot_tag() {
+	# If the repository has no tags, it returns an empty string
+	TAG=$(git ls-remote --quiet --tags --sort="v:refname" | tail -n1 | sed 's/.*\///; s/\^{}//')
+
+	if [ ! -z "$TAG" ]; then
```

Comment:
I know other parts of the script do this, but please use `[ "${TAG}" ]` for variable is set conditionals.

Reply:
Done